### ISSUE TITLE
 Duplicate file ic_launcher.png

### ISSUE THREAD

**Issue by AlexMercier** [navasmdc/MaterialDesignLibrary#304]
..\app\build\intermediates\res\debug\drawable-hdpi-v4\ic_launcher.png: error: Duplicate file.
..\app\build\intermediates\res\debug\drawable-hdpi\ic_launcher.png: Original is here. The version qualifier may be implied.

**Comment by mntgoat** [comment]
Yeah I had the same issue, I had to clone the repo and removed it.

**Comment by ragunathjawahar** [comment]
+1

**Comment by zhouzhiyong619** [comment]
+1
